frontend/src/components/auth/SignUp/SignUpForm.tsx

```diff
@@ -47,7 +47,7 @@ const SignUpForm: React.FC<SignUpFormProps> = ({ setShowSignUp, setEmailName, em
 					return;
 				}
 
-				if (data === false) {
+				if (data === false || !data) {
```

Comment:
Check if "data" is false or falsy is redundant.
```suggestion
				if (!data) {
```